sale_order_line_date/models/sale_order_line.py

```diff
@@ -0,0 +1,40 @@
+# © 2016 OdooMRP team
+# © 2016 AvanzOSC
+# © 2016 Serv. Tecnol. Avanzados - Pedro M. Baeza
+# © 2016 Eficent Business and IT Consulting Services, S.L.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = 'sale.order.line'
+
+    requested_date = fields.Datetime()
```

Comment:
Good to add string to it

Reply:
"requested_date" is converted automatically in "Requested Date" by Odoo. Why add a string?

Reply:
True that, I realised later.